p4-fusion/git_api.cc

```diff
@@ -168,6 +168,11 @@ void GitAPI::InitializeRepository(const bool noCreateBaseCommit)
 	}
 }
 
+git_repository* GitAPI::GetRepoPtr()
+{
+	return m_Repo;
```

Comment:
this feels slightly leaky as the gitAPI controls when or when not the repo is initialized / de-initialized 

I wonder if we could do roughly this in main:

```
labels := p4.Labels()
var labelResults map[name string]LabelResult = forEach(labels, p4.Label())
git.CreateLabels(labelResults, depotPath)
```